src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -134,4 +139,25 @@ public static ReadOnlyConcierge getSampleConcierge() {
                 .map(Tag::new)
                 .collect(Collectors.toSet());
     }
+
+    /**
+     * Returns a PasswordHashList given pairs of keys and values.
+     */
+    public static PasswordHashList getPasswordHashList(Pair<String, String>... pairs) throws IOException {
```

Comment:
Using JavaFX class here is quite strange, since it is intended for GUI usage. How about using a HashMap as argument instead? Seems to do what you are trying to do here.